PWGCF/MultiparticleCorrelations/Core/MuPa-Configurables.h

```diff
@@ -8,22 +8,101 @@
 // In applying this license CERN does not waive the privileges and immunities
 // granted to it by virtue of its status as an Intergovernmental Organization
 // or submit itself to any jurisdiction.
+
 #ifndef PWGCF_MULTIPARTICLECORRELATIONS_CORE_MUPA_CONFIGURABLES_H_
 #define PWGCF_MULTIPARTICLECORRELATIONS_CORE_MUPA_CONFIGURABLES_H_
 
 #include <string>
 
 // Task configuration:
-Configurable<string> cfTaskName{"cfTaskName", "Default task name", "set task name - use eventually to determine weights for this task"};
-Configurable<bool> cfVerbose{"cfVerbose", true, "run or not in verbose mode"};
+Configurable<string> cfTaskName{
+    "cfTaskName", "Default task name",
+    "set task name - use eventually to determine weights for this task"};
+Configurable<string> cfRunNumber{
+    "cfRunNumber", "Some run number",
+    "set run number - TBI temporarily here, this shall be eventually "
+    "automatically obtained from e.g. collision.bc().runNumber()"};
+Configurable<bool> cfVerbose{
+    "cfVerbose", false,
+    "run or not in verbose mode (but not for function calls per particle)"};
+Configurable<bool> cfVerboseForEachParticle{
+    "cfVerboseForEachParticle", false,
+    "run or not in verbose mode (also for function calls per particle)"};
+Configurable<bool> cfUseCCDB{
+    "cfUseCCDB", true,
+    "access personal files from CCDB or from home dir in AliEn"};
 
 // Test0:
-Configurable<bool> cfCalculateTest0{"cfCalculateTest0", false, "calculate or not Test0"};
-Configurable<string> cfLabels{"cfLabels", "/home/abilandz/DatasetsO2/labels.root", "TBI description"};
+Configurable<bool> cfCalculateTest0{"cfCalculateTest0", false,
+                                    "calculate or not Test0"};
+// Configurable<string> cfFileWithLabels{"cfFileWithLabels",
+// "/home/abilandz/DatasetsO2/labels.root", "path to external ROOT file which
+// specifies all labels"}; Configurable<string>
+// cfFileWithLabels{"cfFileWithLabels",
+// "/alice/cern.ch/user/a/abilandz/labels.root", "path to external ROOT file
+// which specifies all labels"};
```

Comment:
Please, don't leave code commented unless it is going to be used in the next iteration

Reply:
Deleted.